javascript/search/binary_search.js

```diff
@@ -0,0 +1,19 @@
+// This function should be implemented, array is assumed to be sorted
+function binarySearch(array, searchItem) {
+    var present = array.indexOf(searchItem);
```

Comment:
You are performing a linear search prior to performing a binary search. This completely defeats the purpose of the function and makes it `Θ(n)`